electrumx/server/env.py

```diff
@@ -27,15 +27,18 @@ class Env(EnvBase):
     # Peer discovery
     PD_OFF, PD_SELF, PD_ON = range(3)
 
-    def __init__(self):
+    def __init__(self, coin=None):
```

Comment:
Please remove the changes to Env; let's just get the package restructuring done.

I don't see the need to change Env given you can just change the environment of the caller.